src/main/java/view/ResultsView.java

```diff
@@ -165,8 +166,9 @@ public static void main(String[] args) {
         ResultsViewModel resultsViewModel = new ResultsViewModel();
         SearchView searchView = SearchUseCaseFactory.create(viewManagerModel, searchViewModel, subViewModel, resultsViewModel, searchDataAccessObject);
 
+        ZipSearchViewModel zipSearchViewModel = new ZipSearchViewModel();
         ResultsUserDataAccessInterface resultsUserDataAccessInterface = new ResultsDataAccessObject();
-        ResultsView resultsView = ResultsUseCaseFactory.create(viewManagerModel, resultsViewModel, searchViewModel, resultsUserDataAccessInterface);
+        ResultsView resultsView = ResultsUseCaseFactory.create(viewManagerModel, resultsViewModel, searchViewModel, zipSearchViewModel, resultsUserDataAccessInterface);
```

Comment:
Modifications to the ResultsView make the ZipSearch use case compatible with it! Also tested the ResultsView with SearchView and works properly.